spaces/S000085/properties/P000102.md

```diff
@@ -0,0 +1,15 @@
+---
+space: S000085
+property: P000102
+value: true
+---
+
+Let $X=\mathbb{R}\backslash\{0\} \cup \{0_\alpha: \alpha\in S\}$ be a real line with multiple origins. Define a semimetric $d:X^2\to[0,+\infty)$ by
+- $d(x,y)=|x-y|$,
+- $d(x,0_\alpha)=d(0_\alpha,x)=|x|$,
+- $d(0_\alpha,0_\beta)=1$,
+- $d(0_\alpha,0_\alpha)=0$,
+
+for any $x,y\in\mathbb{R}\backslash\{0\}$, $\alpha,\beta\in S$, $\alpha\neq\beta$.
+
+Clearly, the sets $B_d(x,\varepsilon):=\{y\in X: d(x,y)<\varepsilon)\}$, for $\varepsilon\in(0,1)$ form a neighbourhood base for $x\in X$.
```

Comment:
Since $x$ and $y$ were used in the previous paragraph to denote elements not equal to the origins, would it be clearer to use different letters here?  Up to you.

Reply:
Now it's a and b in the definition.